content/tutorials/manage-the-rippled-server/configuration/run-rippled-as-a-validator.md

```diff
@@ -100,6 +104,42 @@ On your validator:
 
 
 
+## Connect to the Network Using Public Hubs
+
+This section applies to running a validator that you want to remain an untrusted validator. In this case, it may be more efficient for you to use public hubs to connect your validator to the network than to run stock `rippled` servers to use as proxies.
+
+**Note:** If you are running a trusted validator or a validator that you want to become trusted, [connect your validator to the network using proxies](#connect-to-the-network-using-proxies) instead of public hubs.
```

Comment:
I think the goal of the proxies is more redundancy and better connectivity, not directly becoming 'trusted'.